lib/exzeitable/html/format.ex

```diff
@@ -6,23 +6,31 @@ defmodule Exzeitable.HTML.Format do
   @doc """
   If function: true, will pass the entry to the function of the same name as the entry
   Else just output the field
+  If function: function, will just invoke the function
   This function is tested but coveralls will not register it.
   """
   @spec field(map, atom, map) :: String.t() | {:safe, iolist}
-  def field(entry, key, %{
-        socket: socket,
-        params: %Params{
-          fields: fields,
-          module: module,
-          assigns: assigns
-        }
-      }) do
-    if Kernel.get_in(fields, [key, :function]) do
-      socket = smush_assigns_together(socket, assigns)
+  def field(
+        entry,
+        key,
+        %{
+          params: %Params{
+            fields: fields,
+            module: module
+          }
+        } = root_assigns
+      ) do
+    socket = smush_assigns_together(root_assigns)
 
-      apply(module, key, [socket, entry])
-    else
-      Map.get(entry, key, nil)
+    case Kernel.get_in(fields, [key, :function]) do
+      term when is_boolean(term) and term ->
+        apply(module, key, [socket, entry])
+
+      term when is_function(term) ->
```

Comment:
this allows us to pass a function (FieldFunction.new) to the options, rather than provide an atom that corresponds to a funciton in the module